chord_metadata_service/mcode/mcode_ingest.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from chord_metadata_service.patients.models import Individual
+from .models import *
```

Comment:
in the future these types of star imports are going to cause a linting error, FYI

Reply:
fixed